docs/storage/bestman-install.md

```diff
@@ -0,0 +1,500 @@
+--------------------------------------------------------
+
+The logging directory (`/var/log/bestman2`) has two types of logs - `bestman2.log` and `event.srm.log`.
+
+Log-rotation of `bestman2.log` file is controlled by `/etc/logrotate.d/bestman2` file.
+
+By default, the size of `event.srm.log` log file is set to 50MB within the Bestman code itself.
+
+Left unchanged, `event.srm.log` file counts will keep increasing indefinitely.  Depending on the usage, the number of these files can become high enough to fill up the partition that holds these logs.
+
+There are 3 ways to avoid this -
+
+-   Modify following parameters (commented by default) in the `/etc/sysconfig/bestman2` file
+
+        :::shell
+        # Number of files to keep
+        BESTMAN_EVENT_LOG_COUNT=10
+        # Size of each file in bytes
+        BESTMAN_EVENT_LOG_SIZE=20971520
+
+    The optimal value for these depends on usage of the service.
+
+- Create a directory under a much bigger partition and have a symlink from `/var/log/bestman2` to that directory.
+- Leave the default settings, but have your own custom script that cleans these files according to your needs.
+
+Starting Services
+=================
+
+1. `fetch-crl`
+
+    For RHEL 6:
+
+        :::console
+        [root@client ~] # /usr/sbin/fetch-crl   # This fetches the CRLs 
+        [root@client ~] # /sbin/service fetch-crl-boot start
+        [root@client ~] # /sbin/service fetch-crl-cron start
+
+    For RHEL 7:
+
+        :::console
+        [root@client ~] # /usr/sbin/fetch-crl   # This fetches the CRLs 
+        [root@client ~] # systemctl start fetch-crl-boot
+        [root@client ~] # systemctl start fetch-crl-cron
+
+2. GridFTP
+
+        :::console
+        [root@client ~] # service globus-gridftp-server start
+
+3. Bestman
+
+        :::console
+        [root@client ~] # service bestman2 start
+
+    To start Bestman automatically at boot time
+
+        :::console
+        [root@client ~] # chkconfig bestman2 on
+
+4. Gratia transfer and storage probes
+
+        :::console
+        [root@client ~] # service gratia-gridftp-transfer start
+
+Stopping Services
+=================
+
+1. fetch-crl
+
+    For RHEL 6:
+
+        :::console
+        [root@client ~] # /usr/sbin/fetch-crl   # This fetches the CRLs
+        [root@client ~] # /sbin/service fetch-crl-boot start
+        [root@client ~] # /sbin/service fetch-crl-cron start
+
+    For RHEL 7:
+
+        :::console
+        [root@client ~] # /usr/sbin/fetch-crl   # This fetches the CRLs
+        [root@client ~] # systemctl start fetch-crl-boot
+        [root@client ~] # systemctl start fetch-crl-cron
+
+2. GridFTP
+
+        :::console
+        [root@client ~] # service globus-gridftp-server stop
+
+3. Bestman
+
+        :::console
+        [root@client ~] # service bestman2 stop
+
+4. Gratia transfer and storage probes
+
+        :::console
+        [root@client ~] # service gratia-gridftp-transfer start
```

Comment:
Have we dropped `gratia-xrootd-storage`? If so, you should remove the reference to it on line 342. Also, `start` should be `stop`.